src/Yoh.Text.Json.NamingPolicies.csproj

```diff
@@ -1,6 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <ItemGroup>
-    <PackageReference Include="System.Text.Json" Version="8.0.3" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All"/>
-  </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
`System.Text.Json` is supported by .NET Framework, but isn't included into it. As an example try to compile the following code without referencing `System.Text.Json`:

```
using System.Text.Json;
public class C {
    public void M() {
        JsonSerializer.Serialize("value");
    }
}
```

It will give you a compilation error.

Reply:
I am aware of that hence why the condition is there so that it is only included for net standard hence net framework.

Reply:
Then you shall be able to see compilation errors when running `dotnet build -f net462`. And the reason behind it is that the package shall be referenced without any condition as it was before.

Reply:
why are you trying to build the project against a framework which is not even listed as a target framework for this library?  A client application/library using Net 4.6.2 would use the library as compiled for net standard

Reply:
Ugh, sorry, I saw "framework" in your issue and thought that you wanted, and I also completely missed the description. So, let me check the changes once again.